doc/user/content/manage/operational-guidelines.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Operational guidelines"
+description: "General guidelines for production"
+menu:
+  main:
+    parent: "manage"
+    weight: 4
+    name: "Operational guidelines"
+---
+
+The following provides some general guidelines for production.
+
+## Clusters
+
+### Production clusters for production workloads only
+
+Use production cluster(s) for production workloads only. That is, avoid using
+production cluster(s) to run development workloads or non-production tasks.
+
+### Cluster architecture
+
+{{% best-practices/architecture/three-tier %}}
+
+#### Upsert source consideration
+
+{{% best-practices/architecture/upsert-source %}}
+
+#### Alternatives
+
+Alternatively, if a three-tier architecture is not feasible or unnecessary due
+to low volume or a non-production setup, a two cluster or a single cluster
+architecture may suffice.
+
+{{<tabs>}}
+{{< tab "Two cluster architecture" >}}
+
+{{< best-practices/architecture/two-cluster >}}
+
+{{</ tab >}}
+
+{{< tab "Single cluster architecture" >}}
+
+{{< best-practices/architecture/one-cluster >}}
+
+{{</ tab >}}
+
+{{</ tabs >}}
+
+## Sources
+
+### Scheduling
+
+{{% best-practices/ingest-data/scheduling %}}
+
+### Separate cluster(s) for sources
+
+In production,
+
+- If possible, use a dedicated cluster for [sources](/concepts/sources/); i.e.,
+  avoid putting sources on the same cluster that hosts compute objects, sinks,
+  and/or serves queries.
+
+- Separate upsert sources from other sources. Upsert sources have higher
+  resource requirements (since, for upsert sources, Materialize maintain each
+  key and associated last value for the key as well as to perform
+  deduplication). As such, if possible, use a separate source cluster for upsert
+  sources.
+
+See also [Cluster architecture](#cluster-architecture).
+
+## Sinks
+
+### Separate sinks from sources
+
+Avoid putting sinks on the same cluster that hosts sources.
```

Comment:
Is there a reason for this recommendation?

Reply:
To allow for blue/green deployments -> https://materialize.com/docs/manage/dbt/development-workflows/#configuration-and-initialization.

Reply:
Can add that this is to allow for blue/green.

Reply:
although ... now I think if you only have 2 clusters and your deployment has sinks, route to blue/green might be slightly wonky. Eh ... I'll at least patch to address your comments first.  And then, give a think about what/how to present the sink consideration before merging.

Reply:
Actually, will first patch to address your comments + remove mentions of the sink in the 3-tier vs 2-cluster v 1-cluster blurbs for now.  I'll still give a think about what, if any, should be the next steps before merging.

Reply:
Concluded that will merge with removing mentions of sinks in the section.  And added to my backlog a sinks + blue/green deployment item.